proto/regen/ecocredit/marketplace/v1/tx.proto

```diff
@@ -187,11 +186,10 @@ message MsgBuyDirect {
     // If the marketplace fees end up being greater than this amount, the transaction will fail.
     // If this field is left empty, the marketplace fees will be calculated based on the bid price
     // and quantity and automatically deducted from the buyer's account.
-    // Fees are always paid in the same denomination as the bid price and max fee amount should
-    // be an integer value of that denom.
+    // Fees are always paid in the same denomination as the bid price.
     //
     // Since Revision 3
-    string max_fee_amount = 8 [(cosmos_proto.scalar) = "cosmos.Int"];
+    cosmos.base.v1beta1.Coin max_fee_amount = 8;
```

Comment:
I switched this to a coin so that the ledger formatting is better with on the Ledger with SIGN_MODE_TEXTUAL. Note that it is still not required, although we could decide to require it when there's a non-zero buyer fee for safety on the user's part - although this will require a marketplace app upgrade

Reply:
Is it okay to keep this as not required? @clevinson do you have an opinion?

Reply:
I think it'd be good to have this be required. This way buyers are always consenting to what fees they will pay. In the broader crypto context, users are always consenting to max fees with max gas limits and the like, so I think this makes sense as an approach.

Reply:
I made this required.